package.json

```diff
@@ -30,18 +30,20 @@
     "rebuild-docker-image": "env bash ./scripts/rebuild-docker-image.sh",
     "gzip-and-upload": "env bash ./scripts/gzip-and-upload.sh",
     "build-docs": "echo 'see ./docs-src/README.md'",
-    "test": "jest",
+    "test": "jest test/*.js",
     "integration-test": "NODE_ENV=test ENV=dev jest --config puppeteer.config.js",
+    "smoke-test": "NODE_ENV=test ENV=dev jest ./smoke-test/urls.test.js --config puppeteer.config.js",
+    "smoke-test:ci": "start-server-and-test server http://localhost:4000 smoke-test",
```

Comment:
`start-server-and-test` allows us to bring up and tear down the server in a GitHub action, without making it a background process.